src/celeritas/ext/RootFileManager.cc

```diff
@@ -59,13 +59,15 @@ RootFileManager::make_tree(char const* name, char const* title)
 
 //---------------------------------------------------------------------------//
 /*!
- * Write TFile to disk.
+ * Write TFile to disk and close it.
  */
-void RootFileManager::write()
+void RootFileManager::close()
 {
     CELER_EXPECT(tfile_->IsOpen());
     int write_status = tfile_->Write();
     CELER_ENSURE(write_status);
+    tfile_->Close();
+    CELER_ENSURE(!tfile_->IsOpen());
```

Comment:
I don't think we want this. The ROOT file should be open as long as the RFM exists: we don't want to have a separate "exists and works" and "exists and doesn't work" state. This is one of the fundamantal tenets of good c++ programming: classes should always be in a valid state. And I don't think this class needs the extra complexity of e.g. `ofstream` where you have the ability to close before destruction, separate accessors for validity, etc.

Reply:
Ohh thanks! Good catch. Yes, that was there just for testing and I missed reverting this.